osrf_testing_tools_cpp/src/test_runner/main.cpp

```diff
@@ -39,7 +39,7 @@ int
 main(int argc, char const * argv[])
 {
   std::vector<std::string> args;
-  args.reserve(argc);
+  args.reserve(static_cast<unsigned long>(argc));
```

Comment:
According to https://en.cppreference.com/w/cpp/container/vector/reserve, this takes a `size_type`, which for all practical purposes is `size_t`.  So I think we should cast to that instead.